### ISSUE TITLE
[FEATURE] Make passwords in signup more strong

### ISSUE THREAD

**Issue by MisterMickey** [MilanCommunity/Milan#875]
### Description

Currently there is only one check for password and it's that password must have to be 8 characters long, which is not a good practise , I want to add one more security check to find if any number is entered or not in password and if not it will warn.

### Screenshots

![Screenshot from 2023-06-20 22-59-19](https://github.com/tamalCodes/Milan/assets/93720428/bcf8627c-0c14-4b3e-930b-7f7fd9db9e68)

Assign me under gssoc23

### 🥦 Browser

Brave

### Checklist ✅

- [X] I checked and didn't find similar issue
- [X] I have read the [Contributing Guidelines](https://github.com/tamalCodes/Milan/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (blank for no)

**Comment by tamalCodes** [comment]
Great issue, this will be worked on after #892 is merged into the main. Currently putting this on-hold.

**Comment by tamalCodes** [comment]
Hi @MisterMickey , you can start working on this issue. Do fork all the branches and make a PR to the `Beta` branch as this is related to authentication we will push changes to https://github.com/tamalCodes/Milan/labels/Beta%20Release

### Remember, Good PR makes you a Good contributor !

We at Milan work hard to maintain the structure, and use conventional Pull request titles and commits. Without a proper template for the PR, not following the guidelines and spam might get the pull request closed, or banned.


- Please make sure that you have read our documentation on [Commiting guidelines](https://github.com/IAmTamal/Milan/blob/main/CONTRIBUTING.md#commits-) and [Pull request guidelines](https://github.com/IAmTamal/Milan/blob/main/CONTRIBUTING.md#pull-requests-).

- Once you have done the changes you can `request` a review. If I suggest changes again, do those and `request` a review again.

- Make sure you DO NOT modify the PR template that we have, use it as it is.

- Last but not least, be patient. It takes a while to review the changes and get it merged.
